src/sql/parts/notebook/cellViews/code.css

```diff
@@ -25,6 +25,12 @@ code-component .toolbarIconRun {
 	background-image: url('../media/dark/execute_cell_inverse.svg');
 }
 
+/* overview ruler */
+code-component .monaco-editor.vs .decorationsOverviewRuler,
+code-component .monaco-editor.vs-dark .decorationsOverviewRuler {
```

Comment:
you missed the hc-black (high contrast theme) here. But if it is the same for all themes you can use simply do:
code-component .monaco-editor .decorationsOverviewRuler

Reply:
Thanks, good catch